src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/RazorServiceBase.cs

```diff
@@ -9,22 +9,16 @@
 using Microsoft.AspNetCore.Razor.Language;
 using Microsoft.CodeAnalysis.Razor;
 using Microsoft.CodeAnalysis.Razor.ProjectSystem;
+using Newtonsoft.Json;
 
 namespace Microsoft.CodeAnalysis.Remote.Razor
 {
     internal abstract class RazorServiceBase : ServiceHubServiceBase
     {
         public RazorServiceBase(Stream stream, IServiceProvider serviceProvider)
-            : base(serviceProvider, stream)
+            : base(serviceProvider, stream, GetRazorConverters())
         {
             RazorServices = new RazorServices();
-
-            Rpc.JsonSerializer.Converters.RegisterRazorConverters();
-
-            // Due to this issue - https://github.com/dotnet/roslyn/issues/16900#issuecomment-277378950
-            // We need to manually start the RPC connection. Otherwise we'd be opting ourselves into 
-            // race condition prone call paths.
-            Rpc.StartListening();
```

Comment:
@rynowak @heejaechang, I noticed the issue mentioned in the comment has been fixed. So, are we okay with removing this call?

Reply:
If it works 👍

Reply:
> If it works 👍

The issue seems to be about a race condition so not really sure if that's enough 😄